mozilla/foundation.mozilla.org
# 7380 tips to protect free text

Closes #7380 


**Link to sample test page**: https://foundation-s-7380-tips--u6dvkk.herokuapp.com/en/privacynotincluded/finally-live-program-yourself/


**Steps to test**:

1. Visit the test page above
2. Scroll down and note the new "Tips to protect yourself" section
3. If you would like to test out the editing functionality, please log into the cms using the credentials: (admin2/admin2)
4. Visit https://foundation-s-7380-tips--u6dvkk.herokuapp.com/en/privacynotincluded/economy-fine-theory-including-couple-cover-none/, to test out a page without any tips added. Please not that the "tips to protect yourself" section does not render
5. If everything works as expected, testing is complete! 


## Checklist

**Changes in Models:**
